test/stepDefinitions/validationSteps.js

```diff
@@ -66,12 +66,16 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
   });
 
   When(/^I enter '(.*)' into the '(.*)' field$/, async function (value, fieldType) {
-    await baseSteps.navigateToCreateCasePage()
-    await caseWizardPage.interactWithField(fieldType,value);
+    await baseSteps.navigateToCreateCasePage();
+    await caseWizardPage.interactWithField(fieldType, value);
   });
 
   When(/^I re-enter '(.*)' into the '(.*)' field$/, async function (value, fieldType) {
-    await caseWizardPage.interactWithField(fieldType,value);
+    await caseWizardPage.interactWithField(fieldType, value);
+  });
+
+  When(/^I re-enter '(.*)' into the '(.*)' field with id '(.*)'$/, async function (value, fieldType, id) {
```

Comment:
This looks like a new Step but I don't see it used in the BDD so is it needed any longer or it can be removed? (Also we should try to not to parse IDs in form the BDD but have them coming from the page object level instead)

Reply:
Yes. Will remove it. I was just testing if caseWizardPage.interactWithField(fieldType, value, id) will work.